web/.eslintrc.json

```diff
@@ -16,11 +16,21 @@
         "sourceType": "module"
     },
     "plugins": ["@typescript-eslint", "lit", "custom-elements"],
+    "ignorePatterns": ["authentik-live-tests/**"],
     "rules": {
         "indent": "off",
         "linebreak-style": ["error", "unix"],
         "quotes": ["error", "double", { "avoidEscape": true }],
         "semi": ["error", "always"],
-        "@typescript-eslint/ban-ts-comment": "off"
```

Comment:
Allow variables that are prefixed with an underscore to exist without being used.  Allows for temporary variables and unused arguments in source code.  It's not a perfect solution-- a better one would be to use Typescript's function-specialization typing, but there are  too many places where we elide or ignore some variables in a function's usage that switching over to specialization would be a huge lift.